src/modules/ekf2/params_gnss.yaml

```
@@ -46,6 +46,15 @@ parameters:
       min: 1.0
       unit: SD
       decimal: 1
+    EKF2_GPS_YAW_OFF:
+      description:
+        short: Heading/Yaw offset for dual antenna GPS
+      type: float
+      default: 0.0
+      min: 0.0
+      max: 360.0
+      unit: deg
+      decimal: 3
```

This is mainly a QGC display thing.

```suggestion
      decimal: 1
```